isis/src/base/apps/stretch/stretch_app.h

```diff
@@ -4,6 +4,7 @@
 #include "PvlGroup.h"
 #include "PvlKeyword.h"
 #include "UserInterface.h"
+#include "PvlLog.h"
```

Comment:
Where does PvlLog.h come from?  I can't find it anywhere

Reply:
Not all of the updated apps import PvlLog.h -- when is it necessary / unnecessary?

Reply:
left over from a previous approach I ditched. Should be removed.